### PR TITLE
Feat/#227 loading screen after pressing payment

### PR DESCRIPTION
**Pull request by Sintry1**
### This pull request closes:

✔️ Closes Jira Issue: [TC-](https://treecreate.atlassian.net/browse/TC-)

### Which service(s) does this issue affect:

- [ ] 🛰️ API
- [x] 🛒 webstore
- [ ] 🎓 admin-page

### Types of changes

- [ ] 🐛 Bug fix (a non-breaking change which fixes an issue)
- [x] 🧱 New feature (a non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🤖 CI/CD (a change to the CI/CD pipeline)
- [ ] ⚗️ Refactoring (an update to some already existing code)
- [ ] 💄 Style (Markup, formatting, CSS)

### Documentation Updates

- [x] 📂 No need for updates
- [ ] 📁 Requires an update ( _not done within this issue_ )
- [ ] 🗃️ Has been updated ( _done within this issue_ )

### Testing checklist

- [x] 💪 Manual tests
- [ ] 🔧 Automatic tests

#### Browser compatibility - _**[**Frontend only**]**_

- [ ] Tested on Safari
- [x] Tested on Chrome
- [ ] Tested on Mozilla Firefox

### What changes have been done within this issue?

- Added a 50% opacity div on top of checkout page when isLoading = true
- Added spinner to the div as a visual prompt

### How should this be manually tested?

- Run the webstore on localhost -> add items to basket -> go to checkout -> fill in the form -> press go to payment

### CODE REVIEW

File: apps/webstore/src/app/pages/auth/checkout/checkout.component.html [hunk 1]
```diff
@@ -675,4 +675,11 @@ <h3>{{ alert.type === 'danger' ? 'Error' : alert.type }}!</h3>
             </div>
         </form>
     </div>
+    <!-- <div class="blocker" *ngIf="isLoading">
+        <mat-spinner class="spinner" [diameter]="70">Loading</mat-spinner>
+    </div> -->
```
Comment (Kwandes) [inline apps/webstore/src/app/pages/auth/checkout/checkout.component.html]: Remove the commented out code
```suggestion
```

**Review by Kwandes — CHANGES_REQUESTED** [review]
The `isLoading` variable in the component.ts has to be modified: There is a line that sets it to true after the API response returns a successful response. The problem here is that there are a few seconds between said response is received and the actual redirect happening.

To mitigate this, we simply don't set isLoading to false and let the loading icon keep running while the redirect finishes and quickpay is opened

**Review by Kwandes — APPROVED** [review]
🚀

File: apps/webstore/src/app/pages/auth/checkout/checkout.component.html [hunk 2]
```diff
@@ -675,4 +675,8 @@ <h3>{{ alert.type === 'danger' ? 'Error' : alert.type }}!</h3>
             </div>
         </form>
     </div>
+    <div class="blocker" *ngIf="isLoading">
+        <div class="spinner-border" style="color: var(--green-color)" role="status"></div>
+        <span class="sr-only"></span>
```
Comment (Teodor25) [inline apps/webstore/src/app/pages/auth/checkout/checkout.component.html]: why is this here?

Comment (Sintry1) [reply apps/webstore/src/app/pages/auth/checkout/checkout.component.html]: Could you be a little more specific?
